### PR TITLE
Blacklist and Whitelist

### PR DESCRIPTION
**Pull request by brianteeman**
It is more normal for these to be one word and not two words. I have updated the words2watch list for this https://joomla.github.io/user-interface-text/?user-interface-text/words2watch.md and this simple PR corrects the core strings and comments

### CODE REVIEW

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on 38b62d5391c73a629a04f4f1a0aa52c2c347617d<br /><br />+1 - Thanks

On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11330">issues.joomla.org/joomla-cms/11330</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 38b62d5391c73a629a04f4f1a0aa52c2c347617d<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11330">issues.joomla.org/joomla-cms/11330</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11330">issues.joomla.org/joomla-cms/11330</a>.</sub>

File: libraries/vendor/joomla/filter/src/InputFilter.php [hunk 1]
```diff
@@ -30,15 +30,15 @@ class InputFilter
 	protected static $instances = array();
 
 	/**
-	 * The array of permitted tags (white list).
+	 * The array of permitted tags (whitelist).
```
Comment (wilsonge) [inline libraries/vendor/joomla/filter/src/InputFilter.php]: Can we revert this and make the change in the framework package please :)

Comment (brianteeman) [inline libraries/vendor/joomla/filter/src/InputFilter.php]: Ok I will revert that one. Where do you want me to submit that part

Comment (wilsonge) [inline libraries/vendor/joomla/filter/src/InputFilter.php]: https://github.com/joomla-framework/filter Here please

**Comment by brianteeman** [comment]
Removed the commit to the vendor file

File: libraries/vendor/joomla/filter/src/InputFilter.php (hunk 1 above)
Comment (brianteeman) [inline libraries/vendor/joomla/filter/src/InputFilter.php]: Cant update it there as the bug is not present there and the file is very different anyay

**Comment by wilsonge** [comment]
Thanks :)

File: libraries/vendor/joomla/filter/src/InputFilter.php (hunk 1 above)
Comment (mbabker) [inline libraries/vendor/joomla/filter/src/InputFilter.php]: I'm pretty sure I saw that and felt the way that was written was wrong and
fixed it there at some point.  Either way as long as it's consistently
correct we're good.

**Pull request merged**